### PR TITLE
Feat/profile

### PR DESCRIPTION
**Pull request by kittynglin**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/kittyng.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Kitty Ng
+short_name:
+twitter: @kittyngng1
```
Comment (joshuarrrr) [inline _authors/kittyng.markdown:4]:
```suggestion
twitter: kittyngng1
```

File: _authors/kittyng.markdown [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: Kitty Ng
+short_name:
```
Comment (joshuarrrr) [inline _authors/kittyng.markdown:3]: remove if not needed
```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Please move the new file to the `_authors/osci_2023` directory. Otherwise just a couple small changes needed.

**Comment by dtaivpp** [comment]
Hey to fix DCO can you do the following? 

1. In your local branch, run: git rebase HEAD~4 --signoff
2. Force push your changes to overwrite the branch: git push --force-with-lease origin feat/profile

**Comment by kittynglin** [comment]
Hello!

Thank you for pointing that out! I believe I already did the changes!

Best,

Kitty Ng

**Comment by dtaivpp** [comment]
Closing as we are past deadline and there are outstanding changes.

**Pull request closed without merging**